applications/nrf5340_audio/src/modules/audio_sync_timer.c

```diff
@@ -55,13 +133,29 @@ static int audio_sync_timer_init(void)
 {
 	nrfx_err_t ret;
 
-	ret = nrfx_timer_init(&audio_sync_timer_instance, &cfg, event_handler);
+	ret = nrfx_timer_init(&audio_sync_timer_instance, &cfg, unused_timer_isr_handler);
 	if (ret - NRFX_ERROR_BASE_NUM) {
 		LOG_ERR("nrfx timer init error - Return value: %d", ret);
 		return ret;
 	}
 
+#if !defined(CONFIG_AUDIO_SYNC_TIMER_USES_RTC)
 	nrfx_timer_enable(&audio_sync_timer_instance);
+#endif
+
+#if defined(CONFIG_AUDIO_SYNC_TIMER_USES_RTC)
+	ret = nrfx_rtc_init(&audio_rtc_timer_instance, &rtc_cfg, rtc_isr_handler);
+	if (ret - NRFX_ERROR_BASE_NUM) {
+		LOG_ERR("nrfx rtc init error - Return value: %d", ret);
+		return ret;
+	}
+	IRQ_CONNECT(RTC0_IRQn,
+		    IRQ_PRIO_LOWEST,
```

Comment:
This may be OK, but why is LOWEST pri used?

Reply:
ISR priority is higher than thread priority, therefore we can safely use LOWEST